src/valetudo-map-card.ts

```diff
@@ -1,9 +1,14 @@
-import {FourColorTheoremSolver} from "./ts-lib/colors/FourColorTheoremSolver";
 import * as pako from "pako";
-import {extractZtxtPngChunks} from "./lib/pngUtils";
-import {DEFAULT_CARD_CONFIG, POLL_INTERVAL_STATE_MAP} from "./res/consts";
-import {preprocessMap} from "./ts-lib/mapUtils";
+import { HomeAssistant } from "custom-card-helpers";
```

Comment:
Are these missing in the package.json?

Reply:
Yes, `home-assistant-js-websocket` is a subdependency of `custom-card-helpers`, will add that explicitly to package.json as well. 👍